config.js

```diff
@@ -72,6 +72,7 @@ config = Object.assign({}, config, {
     'brace/mode/html',
     'brace/theme/tomorrow',
     'classnames',
+    'copy-to-clipboard',
```

Comment:
This moves copy clipboard to the vendor dll bundle. The dll bundle is used for files that change rarely. It makes watch mode rebuilds of the primary app code much faster.